ndcube/asdf/resources/schemas/ndcollection-1.0.0.yaml

```diff
@@ -0,0 +1,28 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/ndcollection-1.0.0"
+
+title:
+  Represents the ndcube.ndcollection.NDCollection object
+
+description:
+  Represents the ndcube ndcube.ndcollection.NDCollection object
+
+type: object
+properties:
+  items:
```

Comment:
I'm not sure I like the name "items" here. It feels off. "cubes" also feels weird.